e2eshark/onnx/models/AlexNet_vaiq_int8/model.py

```diff
@@ -42,8 +42,8 @@
 # Post process output to do:
 # sort(topk(torch.nn.functional.softmax(output, 0), 2)[1])[0]
 # Top most probability
-E2ESHARK_CHECK["postprocess"] = [
-    (torch.nn.functional.softmax, [0], False, 0),
-    (torch.topk, [2], True, 1),
-    (torch.sort, [], True, 0),
-]
+# E2ESHARK_CHECK["postprocess"] = [
+#     (torch.nn.functional.softmax, [0], False, 0),
+#     (torch.topk, [2], True, 1),
+#     (torch.sort, [], True, 0),
+# ]
```

Comment:
What's the rationale for commenting out these postprocess steps? Would it be better to just remove the postprocessing code from these files? (as a IREE developer working nearly exclusively with native C/C++ tools, this logic in Python is totally foreign to me, so I'd prefer to leave it out if it isn't strictly needed for testing/development)